### DISCUSSION TITLE
Pop-ups in KDE apps in fullscreen mode

### DISCUSSION THREAD

**Discussion by medwatt** [i3/i3#5492] [Q&A]
I always use the fullscreen mode to give me more real estate. The only problem I face is that some programs, specifically KDE ones like dolphin and okular, have popup dialog boxes that cause program to exit fullscreen mode. For example, let's say I have dolphin in fullscreen mode and I want to delete a file. This opens up a dialog box to confirm the process and results in dolphin exiting fullscreen mode. Is there a way to fix this?

**Comment by joepvd** [comment]
Did you try https://i3wm.org/docs/userguide.html#_popups_during_fullscreen_mode ?
